### PR TITLE
Add ckb script development status rfc

### PR DESCRIPTION
**Pull request by duanyytop**
`sudt` and `anyone_can_pay` scripts of Aggron will be updated later

### CODE REVIEW

**Comment by duanyytop** [comment]
`sudt` and `anyone_can_pay` scripts of Aggron have been updated in https://github.com/nervosnetwork/rfcs/pull/191/commits/b10e5dcd39d5fe8e12c8ea30c22d728c1012b5cd

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+## Abstract
+
+This document shows a series of Nervos CKB system scripts information, including a brief introduction and _code_hash_, _hash_type_, _out_point_(_tx_hash_ and _index_), _dep_type_ in mainnet Lina and testnet Aggron so far.
+
+## Motivation
+
+Nervos Foundation providers a series of Nervos CKB system scripts, including:
+
+- [Default Locks](#default-Locks):
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:20]:
```suggestion
- [Locks](#Locks):
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+  - [_SECP256K1/blake160_](#secp256k1blake160)
+  - [_SECP256K1/multisig_](#secp256k1multisig)
+  - [_anyone_can_pay_](#anyone_can_pay)
+
+- [System Scripts](#system-scripts)
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:26]:
```suggestion
- [Types](#Types)
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+  - [_Nervos DAO_](#nervos-dao)
+
+- [Fundamental Apps](#fundamental-apps)
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:30]:
```suggestion
- [Standards](#Standards)
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+  - [_Simple UDT_](#simple-udt)
+
+To construct transactions with system scripts, the _code_hash_, _hash_type_, _out_point_ and _dep_type_ of system scripts in mainnet Lina and testnet Aggron are needed.
+
+## Default Locks
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:36]:
```suggestion
## Locks
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+| parameter   | value                                                                |
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0x86a1c6987a4acbe1a887cca4c9dd2ac9fcb07405bbeda51b861b18bbf7492c4b` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0x4f32b3e39bd1b6350d326fdfafdfe05e5221865c3098ae323096f0bfc69e0a8c` |
+| `index`     | `0x0`                                                                |
+| `dep_type`  | `dep_group`                                                          |
+
+## System Scripts
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:106]:
```suggestion
## Types
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+| parameter   | value                                                                |
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e` |
+| `hash_type` | `data`                                                               |
+| `tx_hash`   | `0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f` |
+| `index`     | `0x2`                                                                |
+| `dep_type`  | `code`                                                               |
+
+## Fundamental Apps
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:134]:
```suggestion
## Standards
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+  - [_Simple UDT_](#simple-udt)
+
+To construct transactions with system scripts, the _code_hash_, _hash_type_, _out_point_ and _dep_type_ of system scripts in mainnet Lina and testnet Aggron are needed.
+
+## Default Locks
+
+### SECP256K1/blake160
+
+[SECP256K1/blake160](https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#p2ph) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/secp256k1_blake160_sighash_all.c) is a popular script to sign and validate transaction in CKB.
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:40]:
```suggestion
[SECP256K1/blake160](https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#p2ph) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/secp256k1_blake160_sighash_all.c) is the default lock script to verify CKB transaction signature.
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37` |
+| `index`     | `0x1`                                                                |
+| `dep_type`  | `dep_group`                                                          |
+
+### anyone_can_pay
+
+[anyone_can_pay](https://talk.nervos.org/t/rfc-anyone-can-pay-lock/4438) [Source Code](https://github.com/nervosnetwork/ckb-anyone-can-pay) is is a script that can accept any amount of payment.
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:92]:
```suggestion
[anyone_can_pay](https://talk.nervos.org/t/rfc-anyone-can-pay-lock/4438) [Source Code](https://github.com/nervosnetwork/ckb-anyone-can-pay) allows a recipient to provide cell capacity in asset transfer.
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0x4f32b3e39bd1b6350d326fdfafdfe05e5221865c3098ae323096f0bfc69e0a8c` |
+| `index`     | `0x0`                                                                |
+| `dep_type`  | `dep_group`                                                          |
+
+## System Scripts
+
+### Nervos DAO
+
+[Nervos DAO](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0023-dao-deposit-withdraw/0023-dao-deposit-withdraw.md) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/dao.c) is a script and One function of Nervos DAO is to provide an dilution counter-measure for CKByte holders. By deposit in Nervos DAO, holders get proportional secondary rewards, which guarantee their holding are only affected by hardcapped primary issuance as in Bitcoin.
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:110]:
```suggestion
[Nervos DAO](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0023-dao-deposit-withdraw/0023-dao-deposit-withdraw.md) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/dao.c) is the script implements Nervos DAO.
```

File: rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+| `hash_type` | `data`                                                               |
+| `tx_hash`   | `0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f` |
+| `index`     | `0x2`                                                                |
+| `dep_type`  | `code`                                                               |
+
+## Fundamental Apps
+
+### Simple UDT
+
+[Simple UDT](https://talk.nervos.org/t/rfc-simple-udt-draft-spec/4333) [Source Code](https://github.com/nervosnetwork/ckb-miscellaneous-scripts/blob/master/c/simple_udt.c) provides a way to issue custom tokens on Nervos CKB.
```
Comment (janx) [inline rfcs/0024-ckb-script-development-status/0024-ckb-script-development-status.md:138]:
```suggestion
[Simple UDT](https://talk.nervos.org/t/rfc-simple-udt-draft-spec/4333) [Source Code](https://github.com/nervosnetwork/ckb-miscellaneous-scripts/blob/master/c/simple_udt.c) implements the minimum standard for user defined tokens on Nervos CKB.
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+Author: Dylan Duan
+Organization: Nervos Foundation
+Created: 2020-05-21
+---
+
+# CKB System Script List
+
+## Abstract
+
+System scripts are the smart contracts built and deployed by the CKB core team. System scripts complement the function of CKB in a flexible way. System scripts can provide core functions (e.g. [_SECP256k1/blake160_], [_Nervos DAO_]), shared standard implementations (e.g. [_Simple UDT_]) or other auxiliary infrastructure components. This document presents the information of all Nervos CKB system scripts, including a brief introduction and _code_hash_, _hash_type_, _out_point_(_tx_hash_ and _index_), _dep_type_ on mainnet Lina and testnet Aggron.
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:14]:
```suggestion
System scripts are the smart contracts built and deployed by the CKB core team. System scripts complement the function of CKB in a flexible way. System scripts can provide core functions (e.g. [SECP256k1/blake160](#secp256k1blake160) and [Nervos DAO](#nervos-dao)), shared standard implementations (e.g. [Simple UDT](#simple-udt)) or other auxiliary infrastructure components. This document presents the information of all Nervos CKB system scripts, including a brief introduction and _code_hash_, _hash_type_, _out_point_(_tx_hash_ and _index_), _dep_type_ on mainnet Lina and testnet Aggron.
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+
+## Motivation
+
+System scripts are used frequently in dapps, wallets, and other application development. A list of system scripts provides a handy reference to developers.
+
+## List of System Scripts
+
+- [Locks](#Locks):
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:22]:
```suggestion
- [Locks](#Locks)
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+
+  - [_SECP256K1/blake160_](#secp256k1blake160)
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:24]:
```suggestion
  - [SECP256K1/blake160](#secp256k1blake160)
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+  - [_SECP256K1/multisig_](#secp256k1multisig)
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:25]:
```suggestion
  - [SECP256K1/multisig](#secp256k1multisig)
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 15]
```diff
@@ -0,0 +1,152 @@
...
+  - [_anyone_can_pay_](#anyone_can_pay)
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:26]:
```suggestion
  - [anyone_can_pay](#anyone_can_pay)
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 16]
```diff
@@ -0,0 +1,152 @@
...
+
+- [Types](#Types)
+
+  - [_Nervos DAO_](#nervos-dao)
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:30]:
```suggestion
  - [Nervos DAO](#nervos-dao)
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 17]
```diff
@@ -0,0 +1,152 @@
...
+
+- [Standards](#Standards)
+
+  - [_Simple UDT_](#simple-udt)
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:34]:
```suggestion
  - [Simple UDT](#simple-udt)
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 18]
```diff
@@ -0,0 +1,152 @@
...
+
+To construct transactions with system scripts, the _code_hash_, _hash_type_, _out_point_ and _dep_type_ of system scripts in mainnet Lina and testnet Aggron are needed.
+
+## Locks
+
+### SECP256K1/blake160
+
+[SECP256K1/blake160](https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#p2ph) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/secp256k1_blake160_sighash_all.c) is the default lock script to verify CKB transaction signature.
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:42]:
```suggestion
[SECP256K1/blake160](https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#p2ph) ([Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/secp256k1_blake160_sighash_all.c)) is the default lock script to verify CKB transaction signature.
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 19]
```diff
@@ -0,0 +1,152 @@
...
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37` |
+| `index`     | `0x0`                                                                |
+| `dep_type`  | `dep_group`                                                          |
+
+### SECP256K1/multisig
+
+[SECP256K1/multisig](https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#multisig) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/secp256k1_blake160_multisig_all.c) is a script which allows a group of users to sign a single transaction.
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:68]:
```suggestion
[SECP256K1/multisig](https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#multisig) ([Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/secp256k1_blake160_multisig_all.c)) is a script which allows a group of users to sign a single transaction.
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 20]
```diff
@@ -0,0 +1,152 @@
...
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37` |
+| `index`     | `0x1`                                                                |
+| `dep_type`  | `dep_group`                                                          |
+
+### anyone_can_pay
+
+[anyone_can_pay](https://talk.nervos.org/t/rfc-anyone-can-pay-lock/4438) [Source Code](https://github.com/nervosnetwork/ckb-anyone-can-pay) allows a recipient to provide cell capacity in asset transfer.
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:94]:
```suggestion
[anyone_can_pay](https://talk.nervos.org/t/rfc-anyone-can-pay-lock/4438) ([Source Code](https://github.com/nervosnetwork/ckb-anyone-can-pay)) allows a recipient to provide cell capacity in asset transfer.
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 21]
```diff
@@ -0,0 +1,152 @@
...
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0x4f32b3e39bd1b6350d326fdfafdfe05e5221865c3098ae323096f0bfc69e0a8c` |
+| `index`     | `0x0`                                                                |
+| `dep_type`  | `dep_group`                                                          |
+
+## Types
+
+### Nervos DAO
+
+[Nervos DAO](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0023-dao-deposit-withdraw/0023-dao-deposit-withdraw.md) [Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/dao.c) is the script implements Nervos DAO.
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:112]:
```suggestion
[Nervos DAO](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0023-dao-deposit-withdraw/0023-dao-deposit-withdraw.md) ([Source Code](https://github.com/nervosnetwork/ckb-system-scripts/blob/master/c/dao.c)) is the script implements Nervos DAO.
```

File: rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md [hunk 22]
```diff
@@ -0,0 +1,152 @@
...
+| `hash_type` | `data`                                                               |
+| `tx_hash`   | `0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f` |
+| `index`     | `0x2`                                                                |
+| `dep_type`  | `code`                                                               |
+
+## Standards
+
+### Simple UDT
+
+[Simple UDT](https://talk.nervos.org/t/rfc-simple-udt-draft-spec/4333) [Source Code](https://github.com/nervosnetwork/ckb-miscellaneous-scripts/blob/master/c/simple_udt.c) implements the minimum standard for user defined tokens on Nervos CKB.
```
Comment (janx) [inline rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md:140]:
```suggestion
[Simple UDT](https://talk.nervos.org/t/rfc-simple-udt-draft-spec/4333) ([Source Code](https://github.com/nervosnetwork/ckb-miscellaneous-scripts/blob/master/c/simple_udt.c)) implements the minimum standard for user defined tokens on Nervos CKB.
```

**Pull request merged**